guides/Visual_design/Data visualisation/plotly.md

```diff
@@ -0,0 +1,27 @@
+# Plotly
+
+# Introduction
+Plotly from the name it gives clear indication of it what it provides.The logical breakdown of the name tells something which is related to plotting something(i.e graphs charts or visual graphics) isn't ?.A formal description of plotly is described as follows
+
+# Description
+Plotly is one of the data visualization and data analytics tools.It is built on top of data visualization library D3.js, HTML and CSS. It is developed using Python and  Django. It is compatible with a number of languages/ tools: R, Python, MATLAB, Perl, Julia, Arduino. Plotly provides online graphing and statistics tools for individuals and collaborations .It also offers scientific graphing libraries for Python, R, MATLAB, Perl, Julia, and REST.The github repo of plotly is found [here](https://github.com/plotly).
+
+It provides six main products
+### Dash - 
+Dash is a Open Source Python library for creating reactive, Web-based applications
+### plotly.js and plotly.py
+plotly.js and plotly.py are high-level, declarative charting libraries. They ship with over 30 chart types, including scientific charts, 3D graphs, statistical charts, SVG maps, financial charts, and more.
+### falcon-sql-client
+Falcon is a free, open-source SQL editor with inline data visualization.
+### react-plotly.js
+A plotly.js React component from Plotly. The basis of Plotly's React component suite.
+### react-chart-editor
+Customizable React-based editor panel for Plotly charts, Part of Plotly's React Component Suite for building data visualization Web apps and products.
+
+# Get hands on with learning
+Here are few referral links which provide a good documentation for learning
```

Comment:
Can you please add break statement, while providing links.On same lines, it looks cluttered.
Please use this [cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet), to know about how to add bullets.

Reply:
@RN0311 if I add break statement after the word then it  would not  redirect to the specific link .Do you want me to add as such?

Reply:
The links will not be broken. But bulleting is a must here.